app/helpers/structured_data_helper.rb

```diff
@@ -38,27 +38,34 @@ def blog_structured_data(page)
     structured_data("BlogPosting", data)
   end
 
-  def home_structured_data
+  def government_organization_structured_data
     data = {
-      name: "Get Into Teaching",
-      url: root_url,
-      potentialAction: {
-        "@type": "SearchAction",
-        "target": {
-          "@type": "EntryPoint",
-          "urlTemplate": CGI.unescape(search_url(search: { search: "{search_term_string}" })),
-        },
-        "query-input": "required name=search_term_string",
+      "@context": "https://schema.org",
+      "@type": "GovernmentOrganization",
+      "url": root_url,
+      "name": "Get Into Teaching",
+      "description": "Get Into Teaching is the official UK government service that provides comprehensive information and support for individuals considering a teaching career at primary or secondary level. It offers guidance on the qualifications needed, the routes into becoming a primary or secondary school teacher, funding options, and access to one-to-one advice. Get Into Teaching aims to help people become teachers by outlining the steps involved in training and certification, while promoting teaching as a rewarding career.",
```

Comment:
we need to include an email field too for the data to be parsed by google

Reply:
Email is optional according to Google documentation - we shouldn't have to include this?

https://developers.google.com/search/docs/appearance/structured-data/organization